d3a_api_client/utils.py

```diff
@@ -259,6 +262,9 @@ def log_market_progression(message):
                                                          CUSTOMER_WEBSOCKET_DOMAIN_NAME)
 
 
+simulation_id_from_env = os.environ.get("API_CLIENT_SIMULATION_ID", API_CLIENT_SIMULATION_ID)
```

Comment:
Since this is a module constant, it would be great if you could use uppercase letters and put it on the top of the module, after the imports, as recommended by PEP8.